doc/conf.py

```diff
@@ -0,0 +1,262 @@
+# -*- coding: utf-8 -*-
+#
+# ZEO documentation build configuration file, created by
+# sphinx-quickstart on Thu Oct 18 00:16:24 2012.
+#
+# This file is execfile()d with the current directory set to its containing dir.
+#
+# Note that not all possible configuration values are present in this
+# autogenerated file.
+#
+# All configuration values have a default; values that are commented out
+# serve to show the default.
+
+import sys
+import os
+
+# If extensions (or modules to document with autodoc) are in another directory,
+# add these directories to sys.path here. If the directory is relative to the
+# documentation root, use os.path.abspath to make it absolute, like shown here.
+
+sys.path.insert(0, os.path.abspath('.'))
+sys.path.insert(0, os.path.abspath(".."))
+
+# -- General configuration -----------------------------------------------------
+
+# If your documentation needs a minimal Sphinx version, state it here.
+#needs_sphinx = '1.0'
+
+version = '5.2.1.dev0'
```

Comment:
To avoid needing to repeat the version (a step that's usually forgotten, so it quickly grows stale) many projects are now using `pkg_resources` to get the version. [zope.interface example](https://github.com/zopefoundation/zope.interface/blob/7d638c3b149222b164794cac3907c32ab19d2655/docs/conf.py#L18).

Reply:
Nice!